src/main/groovy/com/athaydes/gradle/ceylon/CeylonConfig.groovy

```diff
@@ -20,7 +20,7 @@ class CeylonConfig {
     Boolean verbose = false
     String javaRuntimeDestination
     String entryPoint
-
+    String jarOutput;
```

Comment:
To make it clear this is about the `fatJar` task and be consistent with other properties, I would prefer this to be called `fatJarDestination`.